src/test/java/nextstep/ladder/domain/LinesTest.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.ladder.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalStateException;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class LinesTest {
+
+    @Test
+    public void 사다리_한_줄_생성테스트() {
+        List<Boolean> lines = List.of(true, false, true, false);
+        assertThat(new Lines(lines))
+                .isEqualTo(new Lines(lines));
+    }
+
+    @Test
+    public void 사다리는_연속_되면안된다() {
+        assertThatIllegalStateException().isThrownBy(() -> new Lines(List.of(true, true)));
+    }
+
+    @Test
+    public void 사다리의_이동결과를_반환한다() {
+        Lines lines = new Lines(List.of(true, false, true));
```

Comment:
```suggestion
        Lines lines = new Lines(true, false, true);
```
가변인자를 받는 생성자를 추가하면 위와 같이 구현할 수도 있지 않을까?
다수의 생성자를 지원할 수록 객체를 사용하는 사용자는 편해질 수 있다.